makani/models/noise.py

```diff
@@ -77,15 +77,18 @@ def __init__(
         self.mmax = self.isht.mmax
 
         # generator objects:
+        self.set_rng(seed=seed)
+
+        # store the noise state: initialize to None
+        self.register_buffer("state", torch.zeros((batch_size, self.num_time_steps, self.num_channels, self.lmax_local, self.mmax_local, 2), dtype=torch.float32), persistent=False)
```

Comment:
if se set persistent=False, the state will not be in the checkpoint? is this just for making it simpler to move it from device and back?

Reply:
Yes, the state will not be in the checkpoint but this was the original setting of this line. I have not changed it, only moved it.

Reply:
that sounds good, thanks